data/discord_messages.yaml

```diff
@@ -1,3 +1,1664 @@
+    share the link here.
+
+
+    Image posting is blocked in most channels to discourage people from sharing
+    text as images. Sharing text as images assumes that everybody sees the world
+    as you do, which isn't the case. Some people are colour blind, or have
+    visual impairment that means they can't make sense of an image of text.
+tracker-owntracks:
+  content: >-
+    Remove all reference to the `owntracks` device tracker and [set up the
+    integration](https://home-assistant.io/integrations/owntracks)
+issues:
+  content: >-
+    Want to see what issues are open (or were open and are now closed)?
+
+
+    * [The core (backend)](https://github.com/home-assistant/core/issues) - for
+    HA itself, and integrations
+
+    * [The frontend (UI)](https://github.com/home-assistant/frontend/issues) -
+    for cards and display issues
+
+    * [The Supervisor](https://github.com/home-assistant/supervisor)
+
+    * [HassOS](https://github.com/home-assistant/operating-system/issues)
+
+    * [The
+    documentation](https://github.com/home-assistant/home-assistant.io/issues)
+
+
+    Don't forget to check for any [alerts](https://alerts.home-assistant.io/)
+
+
+    If you think an upgrade broke something then:
+
+    1. Downgrade to the previous version to check if it works correctly there
+
+    2. If it does then change the `logger` settings for that integration to
+    `debug`
+
+    3. Upgrade to the new version again
+
+    4. Look in the logs for errors
+
+    5. Open an issue, following the template provided and including the log
+    information
+llat:
+  content: >-
+    The LLAT, or [Long Lived Access
+    Token](https://www.home-assistant.io/docs/authentication/#your-account-profile)
+    can be found in your [user
+    profile](https://my.home-assistant.io/redirect/profile/). For examples of
+    using it for the API see the [API
+    docs](https://developers.home-assistant.io/docs/en/external_api_rest.html)
+vef:
+  content: |-
+    Help us help you. To quickly explain your environment please specify V/E/F
+
+    - Version - Which version of Home Assistant are you running?
+      Example: 0.84.3, 0.82.0, 0.85.beta.1
+    - Enviroment - How are you running Home Assistant?
+      Example: venv, venv-WSL, docker, hassio, native
+    - Frontend: - Which kind of frontend configuration are you using?
+      Example: auto, manual, yaml, legacy (not lovelace)
+
+    Examples: 0.84.3/venv/yaml
+check:
+  content: >-
+    ***Always*** run the configuration check *command* when you make changes.
+    Don't trust the UI check - it misses some problems.
+
+
+    * HAOS & Supervised use [`ha core
+    check`](https://www.home-assistant.io/common-tasks/os/#configuration-check)
+
+    * Container uses [`docker`
+    commands](https://www.home-assistant.io/common-tasks/container/#configuration-check)
+
+    * Core requires you to [activate the venv
+    first](https://www.home-assistant.io/common-tasks/core/#configuration-check)
+list:
```

Comment:
This one can go

Reply:
We import all or nothing
70-80 % needs adjustments or can be removed.